static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,64 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const GITHUB_CLOUD = ["github.com", "www.github.com"];
+
+/**
+ * Method that checks the validity of the passed URL
+ *
+ * @param {string} inputURL
+ * @returns {boolean}
+ */
+const checkValidUrl = inputURL => {
```

Comment:
Worthwhile calling out that its checking a valid GHE url so `const checkValidGHEUrl` or something of the sort would be nice

Reply:
Renamed to `checkValidGHEUrl`